<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QActuatorWidget</class>
 <widget class="QWidget" name="QActuatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>169</width>
    <height>438</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="layout2">
   <item>
    <widget class="QCheckBox" name="enabled_checkbox">
     <property name="text">
      <string>Enabled</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="static5">
     <property name="title">
      <string>Path:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPlainTextEdit" name="passes_text_edit">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Coordinates of linear passes in an actuator trajectory (x&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;, y&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;, ..., x&lt;span style=&quot; vertical-align:sub;&quot;&gt;n&lt;/span&gt;, y&lt;span style=&quot; vertical-align:sub;&quot;&gt;n&lt;/span&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="layout6">
        <item>
         <widget class="QToolButton" name="clear_button">
          <property name="toolTip">
           <string>Clear</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res/images.qrc">
            <normaloff>:/icons/clear</normaloff>:/icons/clear</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="replace_toggle_button">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Toggle to choose between appending and overwriting new passes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="res/images.qrc">
            <normaloff>:/icons/replace</normaloff>:/icons/replace</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="static6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>16</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="static3">
     <property name="title">
      <string>Shape:</string>
     </property>
     <layout class="QVBoxLayout" name="layout3">
      <item>
       <widget class="QGraphicsView" name="actuator_preview"/>
      </item>
      <item>
       <widget class="QRadioButton" name="coordinates_button">
        <property name="text">
         <string>Outline coordinates</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="coordinates_line_edit">
        <property name="text">
         <string>0 0 10 0 10 10 0 10</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout4">
     <item>
      <widget class="QLabel" name="static4">
       <property name="text">
        <string>Orientation:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="orientation_spinbox">
       <property name="font">
        <font>
         <family>Symbol</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="suffix">
        <string> p</string>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>2.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.050000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout1">
     <item>
      <widget class="QLabel" name="static1">
       <property name="text">
        <string>Speed:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="speed_spinbox">
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999.000000000000000</double>
       </property>
       <property name="value">
        <double>2.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="static2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;tick&lt;span style=&quot; vertical-align:super;&quot;&gt;-1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="allow_spring_crossing_checkbox">
     <property name="text">
      <string>Allow spring crossing</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res/images.qrc"/>
 </resources>
 <connections/>
</ui>
